delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/jwt/ParsedToken.scala

```diff
@@ -33,13 +54,13 @@ object ParsedToken {
     def parseJwt: Either[TokenRejection, SignedJWT] =
       Either
         .catchNonFatal(SignedJWT.parse(token.value))
-        .leftMap(_ => InvalidAccessTokenFormat)
+        .leftMap { e => InvalidAccessTokenFormat(e.getMessage) }
 
     def claims(jwt: SignedJWT): Either[TokenRejection, JWTClaimsSet] =
       Either
         .catchNonFatal(jwt.getJWTClaimsSet)
-        .filterOrElse(_ != null, InvalidAccessTokenFormat)
-        .leftMap(_ => InvalidAccessTokenFormat)
+        .leftMap { e => InvalidAccessTokenFormat(e.getMessage) }
+        .filterOrElse(_ != null, InvalidAccessTokenFormat("No claim is defined."))
```

Comment:
If we are expecting `null` then we could instead say `Either.catchNonFatal(Option(jwt.getJWTClaimsSet))` so the null never escapes